### PR TITLE
[TOOLS-2028] Improvement of the caching for the charts controllers

### PR DESCRIPTION
**Pull request by emannuelalmeida**

### CODE REVIEW

File: kpis/src/main/java/objective/taskboard/controller/FollowUpProgressController.java [hunk 1]
```diff
@@ -23,114 +23,78 @@
...
-    @PostConstruct
-    public void initCache() {
-        cache = cacheManager.getCache(CacheConfiguration.DASHBOARD_PROGRESS_DATA);
+    @CacheEvict
+    @EventListener
+    public void clearCache(SnapshotGeneratedEvent event) {
     }
 
+//    @Caching(
+//        put = {
```
Comment (objective-solutions-user) [inline kpis/src/main/java/objective/taskboard/controller/FollowUpProgressController.java]: BLOCKER This block of commented-out lines of code should be removed. [rule](http://10.44.1.250:9000/coding_rules#rule_key=squid%3ACommentedOutCodeLine)

File: kpis/src/main/java/objective/taskboard/controller/FollowUpProgressController.java [hunk 2]
```diff
@@ -23,114 +23,78 @@
...
+//            @CachePut(key="#projectKey + #zoneId + #projection.get()", condition= "#projection != null"),
+//            @CachePut(key="#projectKey + #zoneId")
+//        }
```
Comment (objective-solutions-user) [inline kpis/src/main/java/objective/taskboard/controller/FollowUpProgressController.java]: BLOCKER This block of commented-out lines of code should be removed. [rule](http://10.44.1.250:9000/coding_rules#rule_key=squid%3ACommentedOutCodeLine)

**Comment by objective-solutions-user** [comment]
SonarQube analysis reported 2 issues
* ![BLOCKER][BLOCKER] 2 blocker

Watch the comments in this conversation to review them.

[BLOCKER]: https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER'

**Comment by emannuelalmeida** [comment]
Another pull request was created for another issue.

**Pull request closed without merging**